aio/aio-proxy/aio_proxy/parameters.py

```diff
@@ -183,3 +183,34 @@ def extract_parameters(
     }
 
     return terms, page, per_page, filters
+
+
+def parse_and_validate_latitude(request):
+    lat = float(request.rel_url.query.get("lat", None))
+    if lat > 90 or lat < -90:
+        raise ValueError("Veuillez indiquer une latitude entre -90° et 90°.")
+    return lat
+
+
+def parse_and_validate_longitude(request):
+    lon = float(request.rel_url.query.get("lon", None))
+    if lon > 90 or lon < -90:
```

Comment:
There is a difference between test and error message : `180 vs 90`

Reply:
I am not sure I understand the question 😅
These commits were an initial trial so I have not finished working on them yet.
I did, however, try to refactor and have one `parse_and_validate_coordinate` function for both `longitude`and `latitude`, but it had too many `if` blocks, so I thought it better to leave it as is.
I also tried to use the `parse_and_clean_parameter` function but they're just too different and it added a layer of complexity instead of simplicity.

Reply:
Sure, sorry for the comments, they were a bit early.
 I will wait for the PR to move from "draft" to "to review"